focus-ios/.swiftlint.yml

```diff
@@ -18,8 +18,10 @@ only_rules: # Only enforce these rules, ignore all others
   # - for_where
   # - force_try
   - implicit_getter
-  # - inclusive_language
-  # - invalid_swiftlint_command
+  - inclusive_language
+  - implicit_getter
+  - inclusive_language
+  - invalid_swiftlint_command
```

Comment:
While I appreciate the want to add `implicit_getter` and `invalid_swiftlint_command` (believe me, I have done this multiple times), there are separate tickets for them and we're trying to get better as a team about tracking done work so we can have tickets be representative of actual work that needs to get done, vs stuff that's no longer relevant. So, we should either: a) make separate PRs for these rules, or b) include the tickets in the PR description. I'd say go route B here, because the changes are so minimal. :)

Reply:
Ooooops,my mistake, I dont understand how I didnt see that.Will fix it ASAP

Reply:
@adudenamedruby fixed it now,hopefully :)